### PR TITLE
JBPM-5993: Stunner - Improvements for canvas animations, SVG shapes, texts and selection.

### PR DESCRIPTION
**Pull request by romartin**
Hey @manstis  @hasys 
Here are some fixes for SVG shapes, animations, selection and a few more things. I had been updating and adding tests missing too. Hope the code base is getting more stable now :)
[Here is the JIRA ticket for it](https://issues.jboss.org/browse/JBPM-5993).
Thanks!

### CODE REVIEW

**Comment by hasys** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
Hey @pefernan , please remember to fix the Jenkins build for the `form-modeler`, lemme know if I can help!. 
Thanks!
CC @manstis @hasys

**Comment by pefernan** [comment]
@manstis the fix: https://github.com/kiegroup/jbpm-form-modeler/pull/111

**Comment by hasys** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
hey @manstis @hasys , can any of you do a quick review for this? so this way @ederign can merge it... I have work on top of this. Thanks!!!

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-lienzo/src/main/java/org/kie/workbench/common/stunner/client/lienzo/canvas/controls/ResizeControlImpl.java [hunk 1]
```diff
@@ -136,39 +140,59 @@ protected void doDisable() {
...
-                    }
-                    canvasHandler.getCanvas().getLayer().draw();
-                }
-            };
-            hasEventHandlers.addHandler(ViewEventType.MOUSE_CLICK,
-                                        clickHandler);
-            registerHandler(element.getUUID(),
-                            clickHandler);
+    @SuppressWarnings("unchecked")
+    void onCanvasElementSelectedEvent(@Observes CanvasElementSelectedEvent event) {
```
Comment (romartin) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-lienzo/src/main/java/org/kie/workbench/common/stunner/client/lienzo/canvas/controls/ResizeControlImpl.java]: hehe yeah it's not necessary, well seen :)

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-lienzo/src/main/java/org/kie/workbench/common/stunner/client/lienzo/canvas/controls/ResizeControlImpl.java [hunk 2]
```diff
@@ -136,39 +140,59 @@ protected void doDisable() {
...
+            if (!hasControlPoints.areControlsVisible()) {
+                showCPs(hasControlPoints);
+            } else {
+                hideCPs(hasControlPoints);
+            }
+        }
+    }
+
+    void CanvasClearSelectionEvent(@Observes CanvasClearSelectionEvent clearSelectionEvent) {
+        checkNotNull("clearSelectionEvent",
```
Comment (romartin) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-lienzo/src/main/java/org/kie/workbench/common/stunner/client/lienzo/canvas/controls/ResizeControlImpl.java]: same as above too :)

**Comment by romartin** [comment]
Hey @hasys , thanks for the review!! Good finding about the resize issue, I have fixed it and added tests as well. 

Please, if all fine now, can you approve it and add a comment to let @ederign know that this is ready to be merged?? :)

Thanks in advance guys!

**Comment by hasys** [comment]
Jenkins execute full downstream build

**Pull request merged**